### ISSUE TITLE
[Urgent]: Parameter Change

### ISSUE THREAD

**Issue by ByungKwanLee** [vllm-project/vllm#16481]
### Your current environment

During training,

For the version 0.8.3.

vllm_model = LLM(model='blah blah',  blah blah)

model = huggingface model (assuming vllm_model and model architecture and the weight name is exactly same)
optimizer = Adam(model.parameters(), blah blah)

for iter in range(training_iterations):
    loss = blah blah()
    loss.backward()
    optimizer.step()

   %% what i wanna do is to change the param of vLLM for the param of updated model params during training iterations.
   %% I don't know how to do it. Please let me know. The below is my imagination
   vllm_model.load_weights(model.parameters())

   

### How would you like to use vllm

I want to run inference of a [specific model](put link here). I don't know how to integrate it with vllm.

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
Please check out the RLHF examples: https://docs.vllm.ai/en/latest/getting_started/examples/rlhf.html
